### DISCUSSION TITLE
Docker daemon either stops abruptly or is not recognized until a restart

### DISCUSSION THREAD

**Discussion by thirumalaicb** [rancher-sandbox/rancher-desktop#1632] [Q&A] [answered]
I am using a **Mac M1 chip**, upgraded the OS to **Monterey** and installed Rancher desktop (**v1.0.1**).  I was able to setup the container.  I followed the [FAQ section](https://docs.rancherdesktop.io/faq) and disabled the Kubernetes cluster as answered for the FAQ question "**I don't need the Kubernetes cluster deployed by Rancher Desktop, how do I disable it to save resources?**"  This causes the docker deamon to either stop abruptly or not recognized causing disconnection to the container from the host.

Have any of you faced/experienced this?

Note: I now have the Kubernetes cluster as-is and currently validating to see if this issue happens.

**Comment by thirumalaicb** [comment]
Update: It did not really matter whether the Kubernetes cluster was enabled/disabled.  This issue still exists.  While running the `kubectl get nodes` gave the result as below, the docker daemon is no longer recognized.
`NAME                              STATUS   ROLES                           AGE   VERSION`
`lima-rancher-desktop   Ready      control-plane,master   24h    v1.22.6+k3s1`

**Comment by jandubois** [comment, marked as answer]
As for stopping and starting Kubernetes, the workaround is not really robust. The new 1.1.0 release coming in the next few days will have a checkbox to enable/disable Kubernetes properly.

The docker socket suddenly disappearing is likely covered by #1274 (the title is misleading, it happens even when the machine is not sleeping). Unfortunately there is still no solution to that problem.

**Reply by thirumalaicb** [reply]
@jandubois, thank you for your response.

Fortunately for me, the docker socket works even after at least 12 hours of sleep and I have tested this multiple times.  However, after the "Reset" is done, the issue happens, and is consistent, after ~23-24 hours.

**Reply by thirumalaicb** [reply]
I see #1274 is tracked for everything that happens for docker socket issue.  I am marking this as closed and would follow the respective ticket for a resolution.
